package.json

```diff
@@ -2,10 +2,14 @@
   "name": "react-native-background-upload",
   "version": "6.5.1",
   "description": "Cross platform http post file uploader with android and iOS background support",
-  "main": "src/index.js",
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "react-native": "src/index",
+  "source": "src/index",
   "scripts": {
+    "build": "tsc && tsc-alias",
     "lint": "yarn lint:ci --fix",
-    "lint:ci": "eslint src example",
+    "lint:ci": "eslint src/** example/**",
```

Comment:
I'm surprised by this one, does `src example` not work to specify these dirs as arguments?}

Reply:
Yeah. I'm guessing they stopped working at some point after some upgrade and nobody updated this

Reply:
I was wrong, the error with the old method was a result of some weird behavior with this version of eslint and non `.js` files. Apparently, the right way to fix it is to add `--ext js,jsx,ts,tsx`